client/src/pages/reports/ReportPeople.js

```diff
@@ -0,0 +1,399 @@
+    {Person.map(sortReportPeople(reportPeople.filter(filterCb)), person =>
+      handleAttendeeRow(person)
+    )}
+  </tbody>
+)
+TableBody.propTypes = {
+  reportPeople: PropTypes.array.isRequired,
+  handleAttendeeRow: PropTypes.func,
+  filterCb: PropTypes.func,
+  enableDivider: PropTypes.bool,
+  showDelete: PropTypes.bool
+}
+TableBody.defaultProps = {
+  reportPeople: []
+}
+
+function sortReportPeople(reportPeople) {
+  return reportPeople.sort((rp1, rp2) => {
+    // primary first, then authors, then alphabetical
+    if (rp1.primary !== rp2.primary) {
+      return rp1.primary ? -1 : 1
+    } else if (rp1.author !== rp2.author) {
+      return rp1.author ? -1 : 1
+    }
+    return (rp1.name || rp1.uuid).localeCompare(rp2.name || rp2.uuid)
+  })
+}
+
+const PrimaryAttendeeRadioButton = ({ person, disabled, handleOnChange }) => (
+  <Radio
+    name={`primaryAttendee${person.role}`}
+    className={`primary${!person.primary ? " inActive" : ""}`}
+    checked={person.primary}
+    disabled={disabled || !person.attendee}
+    onChange={() => !disabled && handleOnChange(person)}
+  >
+    <Label bsStyle="primary">Primary</Label>
+  </Radio>
+)
+
+PrimaryAttendeeRadioButton.propTypes = {
+  person: PropTypes.object,
+  disabled: PropTypes.bool,
+  handleOnChange: PropTypes.func
+}
+const ReportAuthorCheckbox = ({
+  person,
+  disabled,
+  isCurrentEditor,
+  handleOnChange
+}) => (
+  <Checkbox
+    name={`authorAttendee${person.role}`}
+    className={`primary${isCurrentEditor ? " isCurrentEditor" : ""}${
+      !person.author ? " inActive" : ""
+    }`}
+    checked={!!person.author}
+    disabled={disabled || isCurrentEditor}
+    onChange={() => !disabled && handleOnChange(person)}
+  >
+    <Label bsStyle="primary">Author</Label>
+  </Checkbox>
+)
+ReportAuthorCheckbox.propTypes = {
+  person: PropTypes.object,
+  disabled: PropTypes.bool,
+  isCurrentEditor: PropTypes.bool,
+  handleOnChange: PropTypes.func
+}
+const ReportAttendeeCheckbox = ({ person, disabled, handleOnChange }) => (
+  <Checkbox
+    name={`authorAttendee${person.role}`}
+    className={`primary${!person.attendee ? " inActive" : ""}`}
+    checked={!!person.attendee}
+    disabled={disabled}
+    onChange={() => !disabled && handleOnChange(person)}
+  >
+    <Label bsStyle="primary">Attendee</Label>
+  </Checkbox>
+)
+ReportAttendeeCheckbox.propTypes = {
+  person: PropTypes.object,
+  disabled: PropTypes.bool,
+  handleOnChange: PropTypes.func
+}
+
+const AttendeeDividerRow = ({ showDelete }) => (
+  <tr className="attendee-divider-row">
+    <td colSpan={showDelete ? 9 : 8}>
+      <hr />
+    </td>
+  </tr>
+)
+AttendeeDividerRow.propTypes = {
+  showDelete: PropTypes.bool
+}
+
+const Th = ({ hide, text, ...otherProps }) => {
+  return (
+    <th {...otherProps}>
+      <div style={{ minWidth: "89px" }}>{!hide && text}</div>
+    </th>
+  )
+}
+Th.propTypes = {
+  text: PropTypes.string,
+  hide: PropTypes.bool
+}
+
+export default ReportPeople
```

Comment:
ReportPeople tables has a lot of layout and styling issues. Writing single comments would be very difficult to comprehend. So, I suggest changes in the following patch file.

[improve-report-people-table-style.txt](https://github.com/NCI-Agency/anet/files/5427866/improve-report-people-table-style.txt)

Reply:
There was some misalignment with input fields, i changed that a little bit. Can you re check if that is fixed for you as well ?